kernel/packages/shared/profiles/sagas.ts

```diff
@@ -274,7 +274,8 @@ export function* initialLoad() {
   } else {
     let baseCatalog = []
     try {
-      const response = yield fetch(getResourcesURL() + '/default-profile/basecatalog.json')
+      const catalogPath = WSS_ENABLED ? '/default-profile/basecatalog.json' : '/default-profile/basecatalog-wss.json'
```

Comment:
I think you have this backwards :P
```suggestion
      const catalogPath = WSS_ENABLED ? '/default-profile/basecatalog-wss.json' : '/default-profile/basecatalog.json'
```